olio/service/base.go

```diff
@@ -19,16 +19,18 @@ type OlioResourceHandler interface {
 }
 
 type OlioBaseService struct {
-	GinEngine     *gin.Engine
-	server        *network.WebServer
-	daemons       []OlioDaemon
-	coreResources map[string]*olioResources.VersionResource
```

Comment:
please restore this notion of core resources, and add the version and health resources to it

Reply:
??? i think you misunderstood the purpose of this map, its supposed to have key of the type of core resource, value of the actual resource